### ISSUE TITLE
404 Not found

### ISSUE THREAD

**Issue by WeijieChen-MacroAnalyst** [jupyter/nbviewer#938]
**Describe the bug**
Jupyter nbviewer is showing : ```404 : Not Found```

**To Reproduce**
1. Open the line in github```'https://github.com/WeijieChen-MacroAnalyst/Linear_Algebra_With_Python/blob/master/Chapter%208%20-%20Vector%20Space%20and%20Subspace.ipynb'```
2. Copy the link and paste into ```'https://nbviewer.jupyter.org/'```
3. Then nbviewer shows ```404 : Not Found Remote HTTP 404: Chapter 8 - Vector Space and Subspace.ipynb not found among 10 files```

**Expected behavior**
Supposed to be opened by nbviewer.

**Desktop (please complete the following information):**
 - OS: [Windows 7]
 - Browser [Chrome]
 - Version [Version 80.0.3987.132 (Official Build) (64-bit)]

**Additional context**

**Comment by parente** [comment]
It seems to be working today: https://nbviewer.jupyter.org/github/WeijieChen-MacroAnalyst/Linear_Algebra_With_Python/blob/master/Chapter%208%20-%20Vector%20Space%20and%20Subspace.ipynb

This is one of a handful of recent reports about a delay between the file appear on GitHub and nbviewer being able to access it.

**Comment by WeijieChen-MacroAnalyst** [comment]
Thanks , I found that the solution is to add ```?flush_cache=true``` at the end of every url.

**Comment by coderzc** [comment]
How long will the delay be displayed？

**Comment by coderzc** [comment]
> Thanks , I found that the solution is to add `?flush_cache=true` at the end of every url.
It doesn't seem to work well
https://nbviewer.jupyter.org/github/coderzc/machine_learning/blob/master/jupyter/tensorflow/tf_Keras_classification_model.ipynb?flush_cache=true

**Comment by WeijieChen-MacroAnalyst** [comment]
> > Thanks , I found that the solution is to add `?flush_cache=true` at the end of every url.
> > It doesn't seem to work well
> > https://nbviewer.jupyter.org/github/coderzc/machine_learning/blob/master/jupyter/tensorflow/tf_Keras_classification_model.ipynb?flush_cache=true

Yes, I do noticed that sometimes it does not work even if with ```?flush_cache=true```, but when I get back to those links after one or two days, it is usually working.

**Comment by tomerav0** [comment]
I have the same issue. I get error 404 when creating new files in github and trying to open them in nbviewer. 
Github repo is update and using `?flush_cache=true` does not work. Trying to access the notebook directly even after a few hours still gives 404.

The way to make it work is to wait a few minutes and to refresh the folder in repo using nbviewer and only then to I can see my notebook.

**Comment by krinsman** [comment]
I think I have mentioned this in one of the other threads mentioning this same issue, but there seems to be an error in the code where `?flush_cache=false` is actually the command that flushes the cache, whereas `?flush_cache=true` does not. So to anyone else reading this with the same problem, I recommend trying `?flush_cache=false` instead.

**Comment by gkamradt** [comment]
> I have the same issue. I get error 404 when creating new files in github and trying to open them in nbviewer.
> Github repo is update and using `?flush_cache=true` does not work. Trying to access the notebook directly even after a few hours still gives 404.
> 
> The way to make it work is to wait a few minutes and to refresh the folder in repo using nbviewer and only then to I can see my notebook.

@tomerav0 What do you mean "refresh the folder in repo using nbviewer." I have a similar issue and flush_cache=true/false on the folder does not work for me.

How do you refresh it?

**Comment by lxndrarura** [comment]
I had a similar problem after deleting and reloading the same file in the repository. I renamed repository and now nbviewer works correctly.
